### ISSUE TITLE
About Results on IU_Xray dataset

### ISSUE THREAD

**Issue by TFboys-lzz** [Markin-Wang/XProNet#6]
Hi, Jun:
     Thanks for you release the code!!
     I followed the sh file provided in git, but the results I reproduced on the IU_Xray dataset are very different from the results in the paper, did I miss something?
![image](https://user-images.githubusercontent.com/38303428/200827844-4abd8108-40ae-4232-b934-60d66947a81c.png)

**Comment by Markin-Wang** [comment]
> Hi, Jun: Thanks for you release the code!! I followed the sh file provided in git, but the results I reproduced on the IU_Xray dataset are very different from the results in the paper, did I miss something? ![image](https://user-images.githubusercontent.com/38303428/200827844-4abd8108-40ae-4232-b934-60d66947a81c.png)

Hi, thanks for your interest, please refer to [issue 3](https://github.com/Markin-Wang/XProNet/issues/3) to see the details about this. Generally speaking, you may need to tune some hyper-parameters, e.g., learning rate.

**Comment by JiuZhouu** [comment]
hi there does anybody got the similar result in the paper? still could not test it out

**Comment by Eldo-rado** [comment]
> hi there does anybody got the similar result in the paper? still could not test it out

Hello, have you solved it? I also encountered the same problem, and it still did not work after modifying the learning rate.
